### ISSUE TITLE
CheckList of What Should We Do?

### ISSUE THREAD

**Issue by shaoshitong** [shaoshitong/Improrved_LR_Dataset_Distillation#1]
In this work, we will introduce a generalized matching mechanism to large scale dataset condensation. This mechanism including three approaches:  (1) Data Densification (2) Generalized Backbone Matching (3) Generalized Statistics Matching.

What's happening now is that we have achieved 31.4% (+10.4%) highest performance with IPC=10 setting on ImageNet-1k through these three methods without tuning hyperparameters.

Based on this, we should do a lot of necessary ablation experiments to evaluate the effective of these methods, including:

- [x] drop rate and batch size
- [x] loss function (MSE+Cross Entropy or KL?)
- [x] determine which backbone contribute the most (ResNet18, EfficientNet, MobileNet, ShuffleNet)
- [x] the weighting choices of the Data Densification loss function
- [x] data condensation with stronger model and stronger ipc number
- [x] the identify effectiveness of these ideas
